gdcdictionary/schemas/reference_file.yaml

```diff
@@ -0,0 +1,144 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "reference_file"
+title: Reference File
+type: object
+namespace: https://chicagoland.pandemicresponsecommons.org
+category: data_file
+program: '*'
+project: '*'
+description: "Data file containing any type of reference required for a data analysis."
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - created_datetime
+  - updated_datetime
+  - state
+  - file_state
+  - error_type
+
+links:
+  - exclusive: false
+    required: true
+    subgroup:
+      - name: core_metadata_collections
+        backref: reference_files
+        label: data_from
+        target_type: core_metadata_collection
+        multiplicity: many_to_one
+        required: false
+      - name: projects
+        backref: reference_files
+        label: data_from
+        target_type: project
+        multiplicity: many_to_one
+        required: false
+
+required:
+  - type
+  - submitter_id
+  - file_name
+  - file_size
+  - md5sum
+  - data_category
+  - data_type
+  - data_format
+
+uniqueKeys:
+  - [ id ]
+  - [ project_id, submitter_id ]
+
+properties:
```

Comment:
should we add a "vresion" prop, and/or maybe a "description" prop?

Reply:
Created a version property.

Reply:
Created a version property.